app/components/DropZone.jsx

```diff
@@ -3,6 +3,8 @@ import DropzoneComponent from 'react-dropzone-component';
 import FilesActions from './../actions/FilesActions';
 import PropTypes from 'prop-types';
 
+Notification.requestPermission();
```

Comment:
The request for permission should be done in the constructor. Also, if the user has already rejected the permission, stop bugging him/her again.

Reply:
is there any advantage in doing so?

Reply:
We shouldn't write arbitrary code outside the react components. It is error-prone. It should be a part of the component life-cycle.